cypress/pageObjects/basePage.ts

```diff
@@ -0,0 +1,12 @@
+export class BasePage {
+  visit(url?: string): Cypress.Chainable<Cypress.AUTWindow> {
+    const defaultUrl = url ?? "";
+
+    return cy.visit(defaultUrl, {
+      auth: {
+        username: "forome",
```

Comment:
Please no credentials in git. Let's handle that via environment variable. `dotenv` npm package will help.